lib/service/Service.ts

```diff
@@ -120,15 +121,16 @@ class Service extends EventEmitter {
     const { currency } = args;
     const balances = new Map<string, { balance: number, pendingOpenBalance: number }>();
     const getBalance = async (currency: string) => {
-      const cmdLnd = this.lndClients[currency.toUpperCase()];
+      const client = this.lndClients[currency.toUpperCase()]
+        // TODO: support all registered tokens for Raiden
+        || (currency.toUpperCase() === 'WETH' && this.raidenClient);
 
-      if (!cmdLnd) {
-        // TODO: throw an error here indicating that lnd is disabled for this currency
```

Comment:
We might want to keep this todo, but maybe change the message to "indicating that there is no swap client for this currency". I feel like an error is appropriate if we're asked for the channel balance of a currency we don't support.